src/rdb.c

```diff
@@ -94,13 +94,14 @@ void rdbReportError(int corruption_error, int linenum, char *reason, ...) {
     exit(1);
 }
 
-static int rdbWriteRaw(rio *rdb, void *p, size_t len) {
+static ssize_t rdbWriteRaw(rio *rdb, void *p, size_t len) {
     if (rdb && rioWrite(rdb,p,len) == 0)
         return -1;
     return len;
 }
 
 int rdbSaveType(rio *rdb, unsigned char type) {
+    /* convert ssize_t to int would not overflow, because len is 1 */
```

Comment:
```suggestion
```
Is this comment needed?  Many other places have this implicit converting

Reply:
This caller is too close to the change. This comment not required indeed. I will remove this.